clusters/app.ci/quayio-pull-through-cache/quayio-pull-through-cache-us-east-1.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: quayio-pull-through-cache-us-east-1
+    namespace: ci
+  spec:
+    to:
+      kind: Service
+      name: quayio-pull-through-cache-us-east-1
+    tls:
+      termination: Edge
+      insecureEdgeTerminationPolicy: Redirect
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: quayio-pull-through-cache-us-east-1
+    namespace: ci
+  spec:
+    selector:
+      app: quayio-pull-through-cache-us-east-1
+    ports:
+    - port: 5000
+      targetPort: 5000
+    type: ClusterIP
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: quayio-pull-through-cache-us-east-1
+    namespace: ci
+    labels:
+      app: quayio-pull-through-cache-us-east-1
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: quayio-pull-through-cache-us-east-1
+    template:
+      metadata:
+        labels:
+          app: quayio-pull-through-cache-us-east-1
+      spec:
+        volumes:
+        - name: config
+          configMap:
+            name: quayio-pull-through-cache-config
+        - name: exec
+          configMap:
+            name: quayio-pull-through-cache-exec
+        containers:
+        - name: quayio-cache
+          image: docker.io/registry:2.8.1
+          ports:
+            - name: http
+              containerPort: 5000
+          volumeMounts:
+          - name: config
+            mountPath: /etc/docker/registry/
+          - name: exec
+            mountPath: /opt/
+          command: ["/bin/sh"]
+          args: ["/opt/start.sh"]
+          env:
+          - name: REGISTRY_STORAGE_S3_ACCESSKEY
+            valueFrom:
+              secretKeyRef:
+                name: quayio-pull-through-cache-user
```

Comment:
Should we use the `registry-pull-credentials` secret in `ci` so that we only have the one secret to maintain?

```
PULL_USERNAME=$(cat docker-config.json | jq '.auths["quay.io"].auth' -r | base64 -d | cut -d ':'  -f 1)
PULL_PASSWORD=$(cat docker-config.json | jq '.auths["quay.io"].auth' -r | base64 -d | cut -d ':'  -f 2)
```

Reply:
Yes, in the end, why not.